README.md

```diff
@@ -46,7 +46,11 @@ To add an organization using AzureAD authentication, create a database organizat
 that includes an auth_type of `azureadv2` and an auth_config like the following ... 
 
 ```
-{}
+{
+    "TenantID": "edf3cc03-7edf-4299-871a-940bc318789c", 
```

Comment:
```suggestion
    "TenantID": "12345678-abcd-1234-871a-940bc318789c", 
    "ClientSecret": "nice and crazy complicated secret :-)", 
    "ApplicationID": "12345678-abcd-1234-92f6-fffe9f3dfc6d"
```

Reply:
just trying to make these more obviously dummy data

Reply:
not sure if that multi-line suggestion will be incorporated properly, just heads up